mk_verificator/tests/networking/mtu/test_mtu.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+from mk_verificator import utils
+
+
+@pytest.mark.parametrize(
+    "group",
+    utils.get_groups(utils.get_configuration(__file__))
+)
```

Comment:
frankly, that's up to regular tendency to code in some style. I don't think so that there is "special" format for brackets in python